### PR TITLE
21.2 pg_catalog table updates

### PR DESCRIPTION
**Pull request by ericharmeling**
Updated pg_catalog docs to reflect 21.2 updates done by @mnovelodou.

Fixes https://github.com/cockroachdb/docs/issues/11803.
Fixes https://github.com/cockroachdb/docs/issues/11404.
Fixes https://github.com/cockroachdb/docs/issues/11389.
Fixes https://github.com/cockroachdb/docs/issues/11372.
Fixes https://github.com/cockroachdb/docs/issues/11333.
Fixes https://github.com/cockroachdb/docs/issues/11318.
Fixes https://github.com/cockroachdb/docs/issues/11444.

### CODE REVIEW

File: v21.2/pg-catalog.md [hunk 1]
```diff
@@ -67,29 +67,64 @@ PostgreSQL 13 system catalog | `pg_catalog` table
...
+`pg_statio_all_tables` | `pg_statio_all_tables` (empty)
+`pg_statio_sys_indexes` | `pg_statio_sys_indexes` (empty)
+`pg_statio_sys_sequences` | `pg_statio_sys_sequences` (empty)
+`pg_statio_sys_tables` | `pg_statio_sys_tables` (empty)
+`pg_statio_user_indexes` | `pg_statio_user_indexes` (empty)
+`pg_statio_user_sequences` | `pg_statio_user_sequences` (empty)
+`pg_statio_user_tables` | `pg_statio_user_tables` (empty)
 `pg_statistic` | None
-`pg_statistic_ext` | `pg_statistic_ext` (empty)
+`pg_statistic_ext` | `pg_statistic_ext`
```
Comment (mnovelodou) [inline v21.2/pg-catalog.md:122]: This seems empty in 21.2

Comment (ericharmeling) [reply v21.2/pg-catalog.md]: Updated.

**Review by stbof — APPROVED** [review]
LGTM. A few style suggestions:

Line 11 (for accessibility and to match intro ordering with column order): See the table below for a detailed comparison between pg_catalog tables and the PostgreSQL 13 system catalogs. -> See the following table for a detailed comparison between PostgreSQL 13 system catalogs and pg_catalog tables.
Line 129 (imperative verb form): Querying `pg_catalog` tables -> Query `pg_catalog` tables

**Review by mnovelodou — APPROVED** [review]
lgtm

**Pull request merged**